typescriptFixme.ts

```diff
@@ -107,9 +138,9 @@ class Car {
 }
 
 // annotate this array
-var arrayOfAnyType = [1, "string", false];
+var arrayOfAnyType: any[] = [1, "string", false];
 
 // fix the foo object so bar and baz can be added
-let foo = {};
+let foo: { [key: string]: any } = {};
```

Comment:
Don't use `any` and `key` for the type. Use an interface.

Reply:
let foo: partial<foo> = {};
foo.bar =123;
foo.bar ="hello world";

Reply:
yes. commit those changes and push up your code

Reply:
This is not correct. implement an interface for `foo` and annotate foo with it